### PR TITLE
Removed API Key

### PR DESCRIPTION
**Pull request by dhruvin2803**

### CODE REVIEW

File: apps/docs/src/pages/index.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Creator } from "nftoupon";
+import { Arbiter, Creator } from "nftoupon";
```
Comment (AmitMirgal) [inline apps/docs/src/pages/index.tsx]: why did we import Arbiter one?

File: packages/nftoupon/src/Arbiter.tsx [hunk 2]
```diff
@@ -36,34 +50,41 @@ interface ResponsePayload {
...
+  const [transactionType, setTransactionType] = React.useState("");
   const [data, setData] = React.useState<NFTouponPayload>([]);
   const [lockParameter, setLockParameter] = React.useState(true);
+  const [offer, setOffer] = React.useState("");
+  const [expiryDate, setExpiryDate] = React.useState("");
+  const [validation, setValidation] = React.useState("");
+  const [imageURL, setImageURL] = React.useState<any>(
+    "https://djfteveaaqqdylrqovkj.supabase.co/storage/v1/object/public/beta-eatozee-web/nft-free.webp"
+  );
+  let localDate = new Date().toLocaleDateString('fr-CA');
```
Comment (AmitMirgal) [inline packages/nftoupon/src/Arbiter.tsx]: why 'fr-CA' as a locale?
can we make const variable please

File: packages/nftoupon/src/Arbiter.tsx [hunk 3]
```diff
@@ -36,34 +50,41 @@ interface ResponsePayload {
...
   const [details, setDetails] = React.useState({
     id: 0,
-    title: '',
-    description: '',
-    imageUrl: '',
-    status: '',
-    cryptoWalletAddress: '',
-    tokenId: '',
+    title: "",
```
Comment (AmitMirgal) [inline packages/nftoupon/src/Arbiter.tsx]: you can make this object as a DEFAULT one so we can reuse it whenever required

File: packages/nftoupon/src/Arbiter.tsx [hunk 4]
```diff
@@ -72,131 +93,103 @@ export const Arbiter = ({ NFToupon_Key }: Props) => {
...
-
+  const signValidator = async (option: any, ws: WebSocket) => {
+    const result = await fetcher(NFToupon_Key, CARGO_URL, option);
+    setWalletAddress(result?.payload);
+    setTransactionType(result?.tx_type);
+    setLockParameter(false);
+    closeSocket(ws);
+  };
   useEffect(() => {
+    console.log(localDate);
```
Comment (AmitMirgal) [inline packages/nftoupon/src/Arbiter.tsx]: please remove console log 🙏🏽

**Review by AmitMirgal — CHANGES_REQUESTED** [review]

File: packages/nftoupon/src/Arbiter.tsx (hunk 4 above)
Comment (harshilparmar) [reply packages/nftoupon/src/Arbiter.tsx]: Suggestion : We can add webpackplugin to remove console logs. And also can add pre-commit hooks to do prettier and all thing kind of stuff.

Comment (AmitMirgal) [reply packages/nftoupon/src/Arbiter.tsx]: yeah good point... 
we can use husky for that... would you like to implement that if you have a bandwidth... @harshilparmar

File: apps/docs/src/pages/index.tsx (hunk 1 above)
Comment (dhruvin2803) [reply apps/docs/src/pages/index.tsx]: Sorry I forgot to remove that extra import

File: packages/nftoupon/src/Arbiter.tsx (hunk 2 above)
Comment (dhruvin2803) [reply packages/nftoupon/src/Arbiter.tsx]: I thought only 'fr-CA' has yyyy/mm/dd format but I changed it to 'en-CA'.

**Comment by AmitMirgal** [comment]
there is a conflicts @dhruvin2803 could you please resolve it

**Comment by dhruvin2803** [comment]
I have closed this PR because I was working on the main branch and I have now changed it to develop. So created a new PR from develop branch.

**Pull request closed without merging**